build.cmd

```diff
@@ -113,10 +140,8 @@ call :build %__args%
 goto :AfterBuild
 
 :build
-:: *** start WCF Content <WCF uses additional msbuild args> ***
-%_buildprefix% msbuild "%_buildproj%" %_defaultBuildConfig% /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=diag;LogFile="%_buildlog%";Append "/l:BinClashLogger,%_binclashLoggerDll%;LogFile=%_binclashlog%" %__args% %_buildpostfix%
-set BUILDERRORLEVEL=!ERRORLEVEL!
-:: *** end WCF Content ***
+%_buildprefix% msbuild "%_buildproj%" /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=normal;LogFile="%_buildlog%";Append "/l:BinClashLogger,%_binclashLoggerDll%;LogFile=%_binclashlog%" !unprocessedBuildArgs! %_buildpostfix% %OfficialBuildIdArg%
+set BUILDERRORLEVEL=%ERRORLEVEL%
```

Comment:
Is reverting back to % instead of ! correct with EnableDelayedExpansion on?  At one point I thought I found even ERRORLEVEL seemed affected by delayed expansion and required the !. It might be good to verify by injecting a known syntax error temporarily to see how the script handles it.